frontend_build/src/parse_bundle_plugin.js

```diff
@@ -90,6 +90,21 @@ var parseBundlePlugin = function(data, base_dir) {
     ],
   };
 
+  var publicPath, outputPath;
```

Comment:
Move this calculation up, so that we can output publicPath as a defined variable.